manifests/profile/neutron/agent.pp

```diff
@@ -1,5 +1,10 @@
 # The profile to set up a neutron agent
 class openstack::profile::neutron::agent {
   include ::openstack::common::neutron
-  include ::openstack::common::ml2::ovs
+
+  case $::openstack::config::neutron_core_plugin {
+    'plumgrid': { include ::openstack::common::plumgrid }
```

Comment:
Same here using neutron entrypoint names of the plugins instead the whole class name.